web/src/types/error.type.ts

```diff
@@ -1,3 +1,10 @@
 export interface ErrorResponse {
   error: string;
 }
+
+export interface ErrorRequest {
+  error: string;
+  cause: {
+    code?: number;
```

Comment:
Any reason this can't be required?

Reply:
Not all Errors might have a code or cause object. Cause should also be optional in this type

Reply:
I think maybe just `cause` should be optional then, because if there is a `cause`, then there ought to be a code. It never makes sense for there to be a `cause` with no `code`.